etc/tools/runsplitter/src/Runsplitter.cc

```diff
@@ -0,0 +1,151 @@
+/*
+ * Author: Paul Schuetze
+ *
+ * Purpose: Splitting .raw files from eudaq into several files
+ *
+ *
+ *
+ */
+
+
+#include "eudaq/FileReader.hh"
+#include "eudaq/FileWriter.hh"
+#include "eudaq/OptionParser.hh"
+
+#include <iostream>
+#include <string>
+#include <fstream>
+
+using namespace std;
+using namespace eudaq;
+
```

Comment:
It is not in a header, so it is still acceptable, but I prefer not having so (general) using...